app/javascript/src/components/Rondel.vue

```diff
@@ -47,31 +47,19 @@
       >
         <div v-if="game.baseGame === 'imperial'">
           <div>
-            <b>
-              Current Tax Chart Position:
-            </b> {{ game.nations.get(game.currentNation).taxChartPosition }}
+            <b> Tax Chart Position </b> will go from <b>{{ game.nations.get(game.currentNation).taxChartPosition }}</b> to <b>{{ nextTaxChartPosition }}</b>
           </div>
           <div>
-            <b>Next Tax Chart Position:</b> {{ nextTaxChartPosition }}
+            <b> {{ game.currentPlayerName }} </b> would receive {{ nextTaxChartPosition - game.nations.get(game.currentNation).taxChartPosition }}m
```

Comment:
Should we add a `$` in front of the amount here?

Reply:
Ah, that's addressed in a future PR (I think #665 but I'm not at my desk right now) if you don't mind